lib/internal/cluster/master.js

```diff
@@ -119,6 +119,9 @@ function createWorkerProcess(id, env) {
       }
     } else {
       inspectPort = process.debugPort + debugPortOffset;
+      const offset = inspectPort - 65535;
+      if (offset > 0)
+        inspectPort = 1023 + offset;
```

Comment:
I would have written this as:
```js
const [minPort, maxPort] = [1024, 65535];
…
    if (inspectPort > maxPort)
      inspectPort = inpectPort - maxPort + minPort - 1;
```

Reply:
@ChALkeR looks a little bit more readable :)